PythonFDKIntroduction/README.md

```diff
@@ -1,58 +1,71 @@
 # Introduction to fdk-python
 
-This tutorial introduces the 
-[Fn Python FDK (Function Development Kit)](https://github.com/fnproject/fdk-python). 
-If you haven't completed the [Introduction to Fn](../Introduction/README.md)
-tutorial you should head over there before you proceed.
+This tutorial introduces the  [Fn Python FDK (Function Development
+Kit)](https://github.com/fnproject/fdk-python).  If you haven't completed the
+[Introduction to Fn](../Introduction/README.md) tutorial you should head over
+there before you proceed.
 
 This tutorial takes you through the Fn developer experience for building
 Python functions. It shows how easy it is to build, deploy and test
 functions written in Python.
 
 ## What is an FDK?
 
-The Fn Project allows developers to choose the intercommunication protocol format between an application and serverless function. 
-There are currently 3 different formats that Fn defines:
+The Fn Project allows developers to choose the intercommunication protocol
+format between an application and serverless function.  There are currently
+three different formats that Fn defines:
 
-    default: whatever an application sends inside the HTTP request body will be transferred to the function
-    HTTP: the function will receive the full HTTP request
-    JSON: the function will receive a JSON object very similar to CNCF OpenEvents
+* default: whatever an application sends inside the HTTP request body will be transferred to the function
+* HTTP: the function receives the full HTTP request
```

Comment:
remove this line and add:
* cloudevent: the function receives the a CloudEvent object